WordPress/gutenberg
# Simplify vendor scripts handling

## What?

A long time ago, Gutenberg Plugin used to use a lot of vendor scripts (moment, react, lodash...) and for some reason we had implemented a "smart" behavior to use CDN URLs to load these scripts. We also built some tooling to fetch these scripts locally, cache them in `vendor` folder and only load the URLs if the local files are not present. 

This PR changes that behavior and instead just use `node_modules` scripts that are already there and just copy them to the `build` folder in the webpack build. This matches the behavior we implemented later in Core.

## Why?

- First, it's way simpler that way
- Second and most importantly, the plugin right now doesn't require internet connection on first load (to cache the scripts), everything is already there.

## Testing Instructions

1- Remove the "build" and "vendor" folders content 
2- Run `npm run build`
3- Cut your internet connection
4- Check that the editor still loads
5- Bonus point: build a zip by running `npm run build:plugin-zip` (be careful this function might delete some gitignored files) to get a plugin zip and test that zip.
